apis/Google.Cloud.Spanner.ResourceNames.V1/Google.Cloud.Spanner.ResourceNames.V1/DatabaseName.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright 2018, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Are these files copies of the auto-generated versions? If so, it may be good to call that out somewhere.

Reply:
Yes, they are - I'll add a comment.